account_invoice_repair_link/models/repair.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class Repair(models.Model):
+    _inherit = 'repair.order'
+
+    @api.multi
+    def action_invoice_create(self, group=False):
+        Invoice = self.env['account.invoice']
+        res = super().action_invoice_create(group)
```

Comment:
Same as above. You only need to store data in the field invoice_id of the repair.order.